src/Core/Data/RecipeReview.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class RecipeReview
+	{
+
+		public UserProfile Author { get; set; }
+
+		public string Comments { get; set; }
+		
+		public bool AuthorBrewedIt { get; set; }
+
+		public bool? AuthorLikedIt { get; set; }
```

Comment:
Removed the rating property altogether, for now, until we decide on an approach.